history/subscriber/persister.py

```diff
@@ -268,6 +268,35 @@ def on_put(req, resp):
                 'Logging level must be DEBUG, INFO, WARNING, ERROR or CRITICAL!', 'level')
 
 
+def handle_choose_service(config, persister, check):
+
+    messenger = Messenger("Persister", config)
+    messenger.init()
+
+    messenger.create_channel(config.dojot['subjects']['devices'], "r")
```

Comment:
Shouldn't that be inside the first if?

Reply:
I'm going to do the tests, about that, and if you don't have any problem, I'll leave it in if as you suppose